dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -830,8 +830,8 @@ def test_diff(self):
             "median": -1,
             "mode": [[3], [], [2]],
             "median_absolute_deviation": 1,
-            "variance": np.nan,
-            "stddev": np.nan,
+            "variance": [np.nan, 9.473684210526315],
```

Comment:
I am assuming that when `value1` is `nan` and `value2` is numeric, we want to return a pair instead of `value1 - value2` which would be `nan`. Open to thoughts.